Eat_and_Greet/src/comps/SpecificChat/SpecificChatMessage/index.styles.tsx

```diff
@@ -5,8 +5,8 @@ const getFontSize = (size: number) => size / fontScale;
 
 const localStyles = StyleSheet.create({
     SentMessageOuterContainer: {
+        paddingTop: 5,
```

Comment:
Thoughts on just set a paddingVertical to like 7.5? Or is the idea to have the inside be like 2/3 up?

Reply:
This is the padding that effects between two chat messages. I liked this styling more because I wanted the space between the chats and the message typing textbox to be larger than the space between the chats and the top of the screen